docs/basics/networkvariable.md

```diff
@@ -43,11 +42,34 @@ Start               | OnNetworkSpawn
 Also, you should only set the value of a `NetworkVariable` when first initializing it or if it is spawned.  It is not recommended setting a `NetworkVariable` when the associated `NetworkObject` is not spawned.
 :::
 
-
 :::tip
 If you need to initialize other components or objects based on a `NetworkVariable`'s initial synchronized state, then you might contemplate having a common method that is invoked on the client side within the `NetworkVariable.OnValueChanged` callback (if assigned) and `NetworkBehaviour.OnNetworkSpawn` method.
 :::
 
+### Supported Types
+
+:::note
+While `NetworkVariable` supports both managed and [unmanaged](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/constraints-on-type-parameters#unmanaged-constraint)types, managed types do come with an amount of additional overhead. Efforts are made to minimize GC allocations for managed `INetworkSerializable` types (a new value is only allocated if the value changes from `null` to non-`null`), but the ability of a type to be `null` adds additional overhead both in logic (checking for nulls before serializing) and bandwidth (every serialization carries an additional byte indicating whether or not the value is `null`).
+
+Also note that any type which contains a managed type is itself a managed type - so a struct that contains `int[]` is a managed type because `int[]` is a managed type.
+:::
+
+`NetworkVariable` provides support for the following types:
+
+* C# unmanaged [primitive types](../advanced-topics/serialization/cprimitives.md): `bool`, `byte`, `sbyte`, `char`, `decimal`, `double`, `float`, `int`, `uint`, `long`, `ulong`, `short`, and `ushort`, which will be serialized by direct memcpy into/out of the buffer.
+
```

Comment:
Is it just `ushort` that will be serialized by direct memcpy? Or is it all the types listed?

Reply:
If it's the latter, I recommend rewording it to avoid confusion (same with the unmanaged types paragraph below):

> C# unmanaged [primitive types](../advanced-topics/serialization/cprimitives.md) (which are serialized by direct memcpy into/out of the buffer): `bool`, `byte`, `sbyte`, `char`, `decimal`, `double`, `float`, `int`, `uint`, `long`, `ulong`, `short`, and `ushort`.